conf/app.routes

```diff
@@ -2,8 +2,17 @@
 
 ->         /govuk-frontend           govuk.Routes
 ->         /hmrc-frontend            hmrcfrontend.Routes
-GET        /tracking.js              controllers.Assets.at(path="/public", file="tracking.js")
-GET        /tracking-transitional.js controllers.Assets.at(path="/public", file="tracking-transitional.js")
+GET        /tracking.js              controllers.Assets.at(path="/public", file="a.js")
+GET        /tracking/a.js            controllers.Assets.at(path="/public", file="a.js")
+GET        /tracking/b.js            controllers.Assets.at(path="/public", file="b.js")
+GET        /tracking/c.js            controllers.Assets.at(path="/public", file="c.js")
+GET        /tracking/d.js            controllers.Assets.at(path="/public", file="d.js")
+GET        /tracking/e.js            controllers.Assets.at(path="/public", file="e.js")
+GET        /tracking/f.js            controllers.Assets.at(path="/public", file="f.js")
+GET        /tracking/sdes.js         controllers.Assets.at(path="/public", file="sdes.js")
+GET        /tracking/transitional.js controllers.Assets.at(path="/public", file="transitional.js")
```

Comment:
I see we have two endpoints in the routes for transitional, is this deliberate to support people who have already integrated?

Reply:
Yes that is to keep supporting those until we can migrate them.